cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/slice-control.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import Icon from '@ant-design/icons';
+
+import { getCVATStore } from 'cvat-store';
+import { Canvas } from 'cvat-canvas-wrapper';
+import { ActiveControl } from 'reducers';
+import CVATTooltip from 'components/common/cvat-tooltip';
+import GlobalHotKeys, { KeyMapItem } from 'utils/mousetrap-react';
+import opencvWrapper from 'utils/opencv-wrapper/opencv-wrapper';
+import { SliceIcon } from 'icons';
+
+export interface Props {
+    updateActiveControl(activeControl: ActiveControl): void;
+    canvasInstance: Canvas;
+    activeControl: ActiveControl;
+    disabled?: boolean;
+    shortcuts: {
+        SWITCH_SLICE_MODE: {
+            details: KeyMapItem;
+            displayValue: string;
+        };
+    };
+}
+
+function SliceControl(props: Props): JSX.Element {
+    const {
+        updateActiveControl, canvasInstance, activeControl, disabled, shortcuts,
+    } = props;
+
+    const dynamicIconProps =
+        activeControl === ActiveControl.SLICE ?
+            {
+                className: 'cvat-slice-control cvat-active-canvas-control',
+                onClick: (): void => {
+                    canvasInstance.slice({ enabled: false });
+                    updateActiveControl(ActiveControl.CURSOR);
```

Comment:
It seems we already set CURSOR in  `onCanvasSliceDone` or `onCanvasCancel`. Same for join